### ISSUE TITLE
desktop-mime-but-no-exec-code

### ISSUE THREAD

**Issue by tsimonq2** [darkshram/seahorse-caja#5]
Hey there :)

Here's some Lintian errors that you should be aware of relating to the desktop files:
```
W: caja-seahorse: desktop-mime-but-no-exec-code usr/share/applications/mate-seahorse-pgp-encrypted.desktop
N:                                                                                                                                                                                                                                            
N:    The desktop entry lists support for at least one mime type, but does not
N:    provide codes like %f, %F, %u or %U for the Exec key.
N:    
N:    If the application can indeed handle files of the listed mime types, it
N:    should specify a way to pass the filenames as parameters.
N:    
N:    Severity: normal, Certainty: possible
N:    
N:    Check: menu-format, Type: binary
N: 
W: caja-seahorse: desktop-mime-but-no-exec-code usr/share/applications/mate-seahorse-pgp-keys.desktop
W: caja-seahorse: desktop-mime-but-no-exec-code usr/share/applications/mate-seahorse-pgp-signature.desktop
```

**Comment by darkshram** [comment]
Thank you. I have fixed this a few minutes ago.

**Comment by darkshram** [comment]
I have released v1.18.3. Thanks you.
